api/apps/geoprocessing/src/app.module.ts

```diff
@@ -10,7 +10,7 @@ import { PlanningUnitsModule } from '@marxan-geoprocessing/modules/planning-unit
 import { TileModule } from '@marxan-geoprocessing/modules/tile/tile.module';
 import { FeaturesModule } from '@marxan-geoprocessing/modules/features/features.module';
 import { ApiEventsModule } from '@marxan-geoprocessing/modules/api-events/api-events.module';
-import { SurfaceCostModule } from '@marxan-geoprocessing/modules/surface-cost/surface-cost.module';
+import { CostSurfaceModule } from '@marxan-geoprocessing/modules/cost-surface/cost-surface.module';
```

Comment:
❤️ for this change at last 😅 

Sometimes I still have nightmares with SurfaceCosts walking towards me like undead characters, muttering "surface-cost... surface-cost...". Not sure why I never unleashed my naming-things-properly OCD daemon back then. I think none of us was very sure about what the domain logic for all this would be, and naming was likewise confusing in our heads.

Reply:
Alex The Reaper is happy making Andrea the Sensei happy